master/internal/api/actor.go

```diff
@@ -0,0 +1,26 @@
+package api
+
+import (
+	"fmt"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	"github.com/determined-ai/determined/master/pkg/actor"
+)
+
+// ProcessActorResponseError checks actor resposne for errors.
+func ProcessActorResponseError(resp *actor.Response) error {
```

Comment:
@hamidzr  I was under the impression we were going to re-review once you made this change. Oops.

Reply:
We should just not be passing a reference here and then just immediately deref'ing it, especially since `actor.Response` is an interface. Interfaces are pointers already basically in go.

Reply:
oh, https://github.com/determined-ai/determined/pull/1331#issuecomment-699217603 was referring to a commit that was already in the branch when you reviewed I would have asked for a re-review otherwise. do you think this needs fixing? If so I can put up another pr.

Reply:
Yeah, I thought it wasn't in 🤦 . Oops. If you want, open another PR. It's not a huge deal just a little confusing.